sdks/java/io/mqtt/src/test/java/org/apache/beam/sdk/io/mqtt/MqttIOTest.java

```diff
@@ -78,7 +78,7 @@ public void startBroker() throws Exception {
   }
 
   @Test(timeout = 60 * 1000)
-  @Category(RunnableOnService.class)
+  @Category(NeedsRunner.class)
```

Comment:
Same as before, I think there is some value on tagging the tests that use a runner, to enforce validations.